xmake/plugins/project/clang/compile_commands.lua

```diff
@@ -44,8 +44,10 @@ function _translate_arguments(arguments)
         if idx == 1 and is_host("windows") and path.extension(arg) == "" then
             arg = arg .. ".exe"
         end
-        if arg:find("-isystem", 1, true) then
-            arg = arg:replace("-isystem", "-I")
+        if arg:startswith("-isystem-after", 1, true) then
+            arg = "-I" .. arg:sub(13, -1)
+        elseif arg:startswith("-isystem", 1, true) then
+            arg = "-I" .. arg:sub(9, -1)
```

Comment:
It will break `-isystem /xxxx` => `-I /xxxx`

Reply:
It should be okay : 
if the argument is `-isystem`, it start with `-isystem` indeed, and arg:sub will return an empty string.

We can prevent that by adding 
```lua
if arg == "-isystem" or arg == "-isystem-after" then
    arg = "-I"
elseif ...
```